src/state/messages/events/index.tsx

```diff
@@ -65,3 +66,18 @@ export function MessagesEventBusProvider({
     </MessagesEventBusContext.Provider>
   )
 }
+
+export function MessagesEventBusProvider({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  const gate = useGate()
+  const {serviceUrl} = useDmServiceUrlStorage()
+  if (gate('dms') && serviceUrl) {
+    return (
+      <Temp_MessagesEventBusProvider>{children}</Temp_MessagesEventBusProvider>
+    )
+  }
+  return children
+}
```

Comment:
Temp conditional provider, only if DMs and the service URL are available to the user.